airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/cdc/PostgresLsnMapper.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.source.postgres.cdc;
+
+import static io.debezium.connector.postgresql.PostgresOffsetContext.LAST_COMMIT_LSN_KEY;
+import static io.debezium.connector.postgresql.PostgresOffsetContext.LAST_COMPLETELY_PROCESSED_LSN_KEY;
+import static io.debezium.connector.postgresql.SourceInfo.LSN_KEY;
+
+import com.google.common.base.Preconditions;
+import io.airbyte.cdk.components.debezium.DebeziumRecord;
+import io.airbyte.cdk.components.debezium.DebeziumState;
+import io.airbyte.cdk.components.debezium.LsnMapper;
+import io.airbyte.cdk.db.PgLsn;
+import org.apache.kafka.connect.source.SourceRecord;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class PostgresLsnMapper implements LsnMapper<PgLsn> {
```

Comment:
I understand this is just a prototype - but the get PgLsn logic needs to be updated to compare `lsn_commit` values instead of `lsn`/`lsn_proc`

https://github.com/airbytehq/airbyte/pull/35939